legal-api/src/legal_api/resources/v2/admin/reviews.py

```diff
@@ -0,0 +1,51 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""API endpoints for retrieving review data."""
+from http import HTTPStatus
+
+from flask import Blueprint, jsonify
+from flask_cors import cross_origin
+
+from legal_api.core import Filing as CoreFiling
+from legal_api.models import Review, UserRoles
+from legal_api.utils.auth import jwt
+
+
+from .bp import bp_admin
+
+
+@bp_admin.route('/reviews/<int:review_id>', methods=['GET', 'OPTIONS'])
+@cross_origin(origin='*')
+@jwt.has_one_of_roles([UserRoles.staff])
+def get_review(review_id: int):
+    """Return specific review."""
+    review = Review.find_by_id(review_id)
+
+    if not review:
+        return jsonify({'message': 'Review not found.'}), HTTPStatus.NOT_FOUND
+    result = review.json
+
+    # Update the submission date if the status is RESUBMITTED
+    if review.status == 'RESUBMITTED' and review.results:
+        review.submission_date = review.results[0].submission_date
+
+    filing = CoreFiling.find_by_id(review.filing_id)
+    if filing:
+        legder = filing.common_ledger_items(review.identifier, filing.storage)
```

Comment:
Wanted to double check if review.identifier is the correct identifier here

Reply:
nope. identifier in review is the foreign identifier

Reply:
no need to use common_ledger_items func. just create the filingLink

Reply:
identifier (for continuation in, its temp), get it from filing. filing.temp_reg